hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/Source.java

```diff
@@ -33,7 +33,6 @@
  * Represents a source from which we can tail data. Assumes a constructor that takes properties.
  */
 public abstract class Source<T> implements Serializable {
-  private static final Logger LOG = LogManager.getLogger(Source.class);
```

Comment:
seems more like removing unused variable?

Reply:
Yes， there is  an unused variable. I will modify the title.